tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -12525,6 +12529,32 @@ SpirvEmitter::processSpvIntrinsicCallExpr(const CallExpr *expr) {
   return retVal;
 }
 
+SpirvInstruction *SpirvEmitter::processRawBufferLoad(const CallExpr *callExpr) {
+  clang::SourceLocation loc = callExpr->getExprLoc();
+  const clang::Expr *addressExpr = callExpr->getArg(0);
+  SpirvInstruction *address = doExpr(addressExpr);
+
+  const HybridPointerType *bufferType =
+      spvBuilder.getPhysicalStorageBufferType(astContext.UnsignedIntTy);
```

Comment:
Just out of a curiosity, is there any reason that you do not use
```
  const SpirvPointerType *bufferType =
      spvBuilder.getPhysicalStorageBufferType(spvContext.getUIntType(32));
```
?

It is a minor issue, but we prefer `SpirvPointerType` because `const SpirvPointerType *SpirvContext::getPointerType(..)` internally keeps the pointers we already created and reuses it while `const HybridPointerType *SpirvContext::getPointerType(..)` does not.

If there is no special reason, could you please replace it with `SpirvPointerType`?

Reply:
Done.